dirs/dirs.go

```diff
@@ -170,6 +174,10 @@ func SetRootDir(rootdir string) {
 
 	SnapStateFile = filepath.Join(rootdir, snappyDir, "state.json")
 
+	SnapCacheDir = filepath.Join(rootdir, snappyDir, "cache")
```

Comment:
Why not `/var/cache/snapd` instead?

Reply:
not writable on core

Reply:
should we change that?

Reply:
I don't see the current approach to be a problem, tbh.